app/src/main/java/com/example/vybe/Models/VibeEvent.java

```diff
@@ -53,12 +53,12 @@ public VibeEvent() {
      *      This is the longitude coordinate of where a vibe event occurred
      */
 
-    public VibeEvent(Vibe vibe, Date datetime, String reason, SocSit socSit, String id, String image, Double latitude, Double longitude) {
+    public VibeEvent(Vibe vibe, Date datetime, String reason, SocSit socSit, String ID, String image, Double latitude, Double longitude) {
         this.vibe = vibe;
         this.datetime = datetime;
         this.reason = reason;
         this.socSit = socSit;
-        this.id = id;
+        this.ID = ID;
         this.image = image;
         this.latitude = latitude;
         this.longitude = longitude;
```

Comment:
This isn't explicitly defined, but for the VibeEvent constructor, we will always need to call `setOwner` somewhere after. Would it be better to set the owner here? It wouldn't taken it in as a parameter, but just have like `this.owner = mAuth.getCurrentUser().getDisplayName()` or whatever it is that is stored for the owner